src/styles/main.scss

```
@@ -1,3 +1,67 @@
 body {
   margin: 0;
 }
+
+.container {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
+  color: white;
+  font-family: Arial, sans-serif;
+  font-size: 100px;
+  max-width: 1199px;
+  height: 100%;
+}
+
+.box_1 {
+  background-color: rgb(100%, 0%, 0%);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-width: 300px;
+  height: 300px;
+}
```

```suggestion
.box_1 {
  background-color: rgb(100%, 0%, 0%);
  display: flex;
  justify-content: center;
  align-items: center;
  min-width: 300px;
  height: 300px;
}
```
Усі стилі, окрім кольору, дублюються для всіх блоків. Не потрібно повторювати код 6 разів.
Стоври клас `container__block` і пропиши туди спільні стилі.

Також, краще використати container__block:nth-child для того, щоб задати колір кожному, замість того, щоб створювати купу класів типу _2, _3 і т.д.